content/docs/User guides/adding-removing-limits.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Adding or removing limits"
+linkTitle: "Adding or removing limits"
+date: 2019-03-26
+weight: 1
+description: >
+  Add or remove limits.
+---
+
+This guide describes how to add or remove limits, a functionality that allows
+users to specify the maximum number of concurrently running frames 
```

Comment:
that allows you to specify

(prefer the second person as you're speaking to the user)